py/src/ai/h2o/sparkling/ml/params/H2OXGBoostParams.py

```diff
@@ -309,10 +296,12 @@ def getMaxLeaves(self):
         return self.getOrDefault(self.maxLeaves)
 
     def getMinSumHessianInLeaf(self):
-        return self.getOrDefault(self.minSumHessianInLeaf)
+        warnings.warn("The method 'getMinSumHessianInLeaf' is deprecated without replacement and will be removed in 3.32.")
+        return 100
```

Comment:
nit: The method used to return a float value (`100.0`).

Reply:
Thanks!